src/doxy/new_header.html

```diff
@@ -19,8 +20,19 @@
     <link href="$relpath^$stylesheet" rel="stylesheet" type="text/css" />
     <link rel="shortcut icon" href="$relpath^favicon.ico" type="image/x-icon" />
     $extrastylesheet
+    <script>
+      var [projectbrief, projectrepo] = '$projectbrief'.split(' PROJECT_REPO=')
```

Comment:
see comment above.  This is a very nice workaround but I'm concerned about the complexity and the dependency we will be placing on the existence of a GIT repo